tox.ini

```diff
@@ -1,10 +1,9 @@
 [tox]
 isolated_build = true
-envlist = py{36,37,38,39}-django{22,30,31,32}
+envlist = py{37,38,39}-django{22,30,31,32}
```

Comment:
Can we also try Django 4 here as well? If it doesn't work without further changes don't worry about it, but I think it wasn't a huge change so it might Just Work.